Bukkit/src/main/java/com/plotsquared/bukkit/managers/BukkitWorldManager.java

```diff
@@ -48,13 +49,16 @@ public void initialize() {
     @Override
     public @Nullable World handleWorldCreation(@NonNull String worldName, @Nullable String generator) {
         this.setGenerator(worldName, generator);
-        final WorldCreator wc = new WorldCreator(worldName);
-        wc.environment(World.Environment.NORMAL);
-        if (generator != null) {
-            wc.generator(generator);
-            wc.type(WorldType.FLAT);
+        if (!FoliaSupport.isFolia()) {
+            final WorldCreator wc = new WorldCreator(worldName);
+            wc.environment(World.Environment.NORMAL);
+            if (generator != null) {
+                wc.generator(generator);
+                wc.type(WorldType.FLAT);
+            }
```

Comment:
There's an issue within the default branch preventing world generation atm, so this can't be tested and verified safely.

Reply:
Folia have not yet a API for create worlds. Before we run in issues, I removed that option for folia